staff/fede-gomez/app-template/profile.html

```diff
@@ -0,0 +1,26 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Profile</title>
+</head>
+<body>
+    <header>
+        <h1>Website</h1>
+    </header>
+    <nav>
+            <a href="./home.html">Home</a>
+            <a href="news.html">News</a>    
+            <a href="profile.html">Profile</a>    
+            <a href="about.html">About</a>    
+            <a href="contact.html">Contact</a>    
+            <a href="register.html">Register</a>    
+            <a href="login.html">Login</a>       
+    </nav>
+    <section>
+        <h2>Profile</h2>
```

Comment:
Nothing else inside this section? Only profile header?

Reply:
You are right. I'll add a "lorem ipsum" :)